sparse/dok.py

```diff
@@ -377,7 +343,7 @@ def todense(self):
         """
         result = np.zeros(self.shape, dtype=self.dtype)
 
-        for c, d in six.iteritems(self.data):
+        for c, d in self.data.items():
```

Comment:
I'm +1 on this change, even if it causes more memory issues in Python 2.

Reply:
With `from builtins import dict` it won't, unless the user purposefully passes an old-style array. :-)